plutus-chain-index/notes/ConnectingToTheTestnet.md

```diff
@@ -0,0 +1,37 @@
+# How to connect to a testnet
+
+## Get `[cardano-node](https://github.com/input-output-hk/cardano-node)` onto the path
+
+The easiest way of doing this is to enter a `nix-shell` then build the node and use `cabal exec` to run it.
```

Comment:
am I running this in the plutus repository or in the cardano-node one?

Reply:
Updated the document with the answer.